fedmsg.d/example-config.py

```diff
@@ -1,5 +1,5 @@
 import socket
-hostname = socket.gethostname()
+hostname = socket.gethostname().split('.')[0]
```

Comment:
The change on this line may be confusing to others, but it is correct.  I left it out on accident when I originally committed the file.